src/main/resources/db/migration/COMMON/V1_0_0_202301100900__initial_schema.sql

```diff
@@ -1,7 +1,37 @@
 CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
 
-CREATE TABLE books
+CREATE TABLE roles
+(
+    id UUID    NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
+    name VARCHAR(50) NOT NULL
+);
+
+CREATE TABLE users
 (
-    id   UUID         NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
-    name VARCHAR(255) NOT NULL
+    id   uuid         NOT NULL PRIMARY KEY DEFAULT uuid_generate_v4(),
+    username VARCHAR(50) NOT NULL,
+    password VARCHAR(50) NOT NULL,
+    firstName VARCHAR(100),
+    lastName VARCHAR(100),
+    avatar bytea,
```

Comment:
```suggestion
    avatar VARCHAR(1024),
```

Mấy cái tên kiểu dữ liệu thì cho viết hoa hết đi

Reply:
DONE